### PR TITLE
adds mud pie

### PR DESCRIPTION
**Pull request by ghost**
I had a brief look through the coderbase and found that the chemical known as Toxin is actually somewhat hard to come by unless you abuse medvends or make the dankest of the dank ambrosia. As toxin is a requirement to properly farm cockroaches, I thought I'd give it a go at adding some more simple foods you can make from scraps in maintenance.

mud_pie
Mud pie, generally not good to eat unless you're a good cook, and then it's even worse to eat.

:cl:
* rscadd: Adds mud pie. Made from 2 lots of sand and 25 units of water. Perfect for throwing at the head of security

### CODE REVIEW

**Comment by Arthurdentist** [comment]
Could you make it a mud cake instead? 
http://i.dailymail.co.uk/i/pix/2008/08/09/article-1043125-00082E7200000578-955_468x286.jpg

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -3724,17 +3724,17 @@
 	bitesize = 2
 
 /obj/item/weapon/reagent_containers/food/snacks/gigapuddi/happy
-	desc = "A large crème caramel made with extra love"
+	desc = "A large cr�me caramel made with extra love"
```
Comment (PJB3005) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Uhm, what changed in these lines?

Comment (ghost) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: The accented e changed to a A:: when I added in the crème de la pire in the mud pie description, so tried to change them back, seems they've gone weird

**Comment by Shadowmech88** [comment]
Those are obviously mud cookies.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -4971,4 +4971,23 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/bleachkipper/New()
 	..()
-	reagents.add_reagent(FISHBLEACH, 1)
\ No newline at end of file
+	reagents.add_reagent(FISHBLEACH, 1)
+
+/obj/item/weapon/reagent_containers/food/snacks/pie/mudpie
+	name = "mud pie"
+	desc = "while not looking very appetising, looks like somebody had fun making it."
```
Comment (Shadowmech88) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Need to capitalize `while`.

Comment (Shadowmech88) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Also you misspelled appetizing.

Comment (ghost) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: heckf

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (PJB3005) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: What program are you even using to edit code?

**Comment by PJB3005** [comment]
Seems like you managed to screw up the encoding on the file and you transformed it into Windows-1252.

If you're using DM to code, switch to an editor like Atom or VS Code that was made in the last decade please.

**Comment by ghost** [comment]
forgot I had atom installed on my laptop. Changes made

**Comment by Blithering** [comment]
Can't you get toxin from burned messes?

**Comment by ghost** [comment]
Yeah but that's not as fun as throwing a pie made of mud at the head of security

**Review by ComicIronic — APPROVED** [review]

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**